site/src/comparison.rs

```diff
@@ -1338,13 +1330,13 @@ mod tests {
                 (Category::Primary, 1.0, 3.0),
             ],
             r#"
-|            | mean[^1] | max | count[^2] |
-|:----------:|:--------:|:---:|:---------:|
-| Regressions ❌ <br /> (primary) | 146.7%   | 200.0% | 3         |
-| Regressions ❌ <br /> (secondary) | N/A      | N/A | 0         |
-| Improvements ✅ <br /> (primary) | N/A      | N/A | 0         |
-| Improvements ✅ <br /> (secondary) | N/A      | N/A | 0         |
-| All ❌✅ (primary) | 146.7%   | 200.0% | 3         |
+|  | mean[^1] | max | count[^2] |
```

Comment:
Just FYI, the alignment/padding here was added on purpose, so that the table is better aligned and readable in raw text formats (=e-mails).

Reply:
Context: https://github.com/rust-lang/rustc-perf/issues/1310 and https://github.com/rust-lang/rustc-perf/pull/1312

Reply:
I have removed the commit that removed the padding, and added a commit with a comment explaining why it's there.

Reply:
Thanks, I really should have added a comment there before.